spring-cloud-starter-stream-common-cdc-debezium/spring-cloud-starter-stream-common-cdc-debezium-core/src/main/java/org/springframework/cloud/stream/app/cdc/common/core/CdcStreamConfiguration.java

```diff
@@ -92,4 +121,24 @@
 			});
 		};
 	}
+
+	/**
+	 * The default implementation effectively ignores the tombstone messages.
+	 * For example this is the behaviour in case of RabbitMQ binder.
+	 */
+	@Bean
+	@ConditionalOnMissingBean(CdcTombstoneConfiguration.TombstoneSupport.class)
+	public CdcTombstoneConfiguration.TombstoneSupport nullTombstoneHelper() {
+		return new CdcTombstoneConfiguration.TombstoneSupport() {
```

Comment:
That's something too complicated. 
I would load a `KafkaNull` instance into some `static` property if it is present and would initialize it with `null` otherwise.
So, your code above would be just like this:
```
if (cdcJsonPayload == null) {
	cdcJsonPayload = KAFKA_NULL;
}
```

Reply:
Actually if you remember i did implement such code (using reflection).  But static code using reflection doesn't play well for testing.  
Hit the wall while writing tests with the `ApplicationContextRunner().withClassLoader(new FilteredClassLoader(KafkaNull.class))`.  While the test context created like this will exclude the KafkaNull.class, the static implementation would still load it and show it as present on the classpath.